### PR TITLE
Hello-Microverse-PR

### PR DESCRIPTION
**Pull request by laurianerzb**
#This PR is for Microverse hello project

    #Added index.html
    #Added styles.css
    #Added .gitignore
    #Modified README.md
    #Added test.md into .gitignore
    #Set up and run linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+# Project Name
```
Comment (BertrandConxy) [inline README.md]: - [ ] Good job so far 👏. Please add the specific project name  i.e: `Hello Microverse`. This makes your README more specific

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        Hello Microverse Project
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
```
Comment (BertrandConxy) [inline README.md]: - [ ] Please add information in the `Getting started` section as it is among the important section in README. This is where you add procedures to clone and use your project locally 😃 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @laurianerzb , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 
- correct flow is used
## Required Changes ♻️

- [ ] Your Pull request description is not very informative. Normally, this is the place to describe briefly what you have done in the branch. It is better when they are listed
  ```
  Like: - Add inde.html
            - Add styles, etc
  ```
  so please edit your PR description 😃 🏃 

- [ ] It looks like you haven't added the `test.md`  and then,  add it into `.gitignore` file so that it is not tracked by git 😃 . Please add it and make sure it is not pushed on Github 😎 
- [ ] Please set up and run linters and make sure they are passing (i.e: All in green color). You can look at this [Microverse documentation](https://github.com/microverseinc/linters-config/tree/master/html-css)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by laurianerzb** [comment]
@BertrandConxy should I push the .github/workflows folder to GitHub also?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# Project Name
```
Comment (Felix45) [inline README.md]: Please consider editing the section titled `Project Name` on your readme to `Hello Microverse` The title of the project usually features prominently at the top of the [Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) please see the screenshots below for more details

## Change this section  of Readme
|Screenshot from 2022-09-21 02-12-02 |

## After changing it should look like this
Screenshot from 2022-09-21 02-10-02|

File: .gitignore [hunk 4]
```diff
@@ -102,3 +103,7 @@ dist
 
 # TernJS port file
 .tern-port
+package-lock.json
+package.json
+.stylelintrc.json
```
Comment (Felix45) [inline .gitignore]: Please note that adding `package.json`, `package-lock.json`, and `stylelintrc.json` to the `.gitignore` file is not considered good practice, kindly remove them from the `.gitignore` file.

- In group projects, your teammates will rely on the `package.json` and `package-lock.json` to install the shared dependencies needed to set up your group project.

- The `stylelintrc.json` file will also be used to check for linter errors in your stylesheets by your team, thus it's important to push it online.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @laurianerzb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights :heavy_check_mark: 
- Good commit messages
- Application is working
- Descriptive pull request summary


 ## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
[Optional]
Please consider updating the title of this pull request to a more descriptive title, a good pull request title should be descriptive of what the pull request is about.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @laurianerzb      , 

Your project is complete .  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**